test/common/stats/thread_local_store_test.cc

```diff
@@ -39,6 +39,19 @@ class StatsThreadLocalStoreTest : public testing::Test {
     store_->addSink(sink_);
   }
 
+  /**
+   * Finds a stat in a vector with the given name.
+   * @param name the stat name to look for.
+   * @param v the vector of stats.
+   * @return the stat
+   */
+  template <typename T> T findByName(const std::string& name, const std::vector<T>& v) {
```

Comment:
The use of this is very similar to what we already do here: https://github.com/envoyproxy/envoy/blob/master/test/test_common/utility.h#L138. Is it worth generalizing both uses to use the same utility code?

Reply:
Yes, done. Good catch.